systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1154,49 +1154,54 @@ private void testDockerImagesForKafkaCluster(String clusterName, int kafkaPods,
     @Test
     void testManualTriggeringRollingUpdate() {
         String coPodName = kubeClient().listPods("name", "strimzi-cluster-operator").get(0).getMetadata().getName();
-        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 1).done();
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 1, 1).done();
```

Comment:
Can this face the same issues if the GC logging test with the failovers? Single node ZK cluster?

Reply:
I don't think so, cause kafka is rolled before ZK, at least I didn't hit it during manual execution of the test

Reply:
But I think someone can easily fall into this trap later by extending the test or reordering it. And it might not be so easily picked up in the review.